### DISCUSSION TITLE
drop down on edit/create

### DISCUSSION THREAD

**Discussion by Aissi17** [aminalaee/sqladmin#436] [Q&A] [answered]
Is there an implementation for using a dropdown menu to select a items (foreign key)  from another model ?

**Comment by aminalaee** [comment]
Hi,
Can you give an example?
I think this already works out of the box.

**Reply by Aissi17** [reply]
I have a user table with a foreign key that references another table for companies. When creating a form to add a new user, I want to include a dropdown menu in the "company" input field, which should contain a list of all existing companies. How can I accomplish this?

**Reply by aminalaee** [reply, marked as answer]
Isn't that already working out of the box?
Can you share a sample code?

**Reply by anton-petrov** [reply]
How to apply this out of the box? I am also trying to replace list with drop down for foreign keys in edit mode.

**Reply by aminalaee** [reply]
If you have a relationship defined on your model, it will automatically show a list of that model in create/edit page .
I guess you don't have a relationship, right?

**Reply by anton-petrov** [reply]
Yes, it shows a *Listbox*, but personally I want to replace it with *Dropdown List*. Using drop down list is more logical and user friendly in case of foreign keys.

**Reply by anton-petrov** [reply]
![image](https://github.com/aminalaee/sqladmin/assets/2428315/49419a59-ff01-4bbb-88b9-cdee782684d7)

**Reply by aminalaee** [reply]
@anton-petrov 
I have to admit my knowledge of UI is really limited :smile: 
 I think the closest thing to getting the drop-down is using `form_ajax_refs` which will render `select2` jquery component and also enable server-side search of foreign keys, so I think you achieve two things with that.

**Reply by anton-petrov** [reply]
Ok!) In addition, how do you think, maybe it's time to update bundled tabler version and other js/css stuff?

**Reply by aminalaee** [reply]
Yeah sure, feel free to go for it, otherwise I can do it, I see they are now on beta version 17, and we are on version 5.
